src/components/SingleBubble/SingleBubble.mdx

```diff
@@ -3,10 +3,39 @@ title: Single bubble
 menu: Components
 ---
 
-import { PropsTable } from 'docz'; 
+import { Playground, PropsTable } from 'docz'; 
 import SingleBubble from './SingleBubble';
 
 # Single bubble
 SVG React component that will render a single bubble shape when given a height, width and style.
 
-<PropsTable of={SingleBubble} />
\ No newline at end of file
+<PropsTable of={SingleBubble} />
+
+
+<Playground>
+  <svg width="200" height="50" viewBox="0 0 200 50">
+    <g 
+      style={{
+        transform: 'scale(1, -1)',
+        transformOrigin: '50% 50%',
+      }}
+    >
+      <rect 
+        width="200" 
+        height="50" 
+        style={{
+          fill: '#f6f6f6', 
+          strokeWidth: 0,
+          stroke:'rgb(0,0,0,0)'
+        }}
+      />
+      <SingleBubble 
+        x={0} 
+        width={50} 
+        height={30} 
```

Comment:
Demo bubble is quite small: https://deploy-preview-56--iiif-timeliner.netlify.com/docs/single-bubble-single-bubble

Reply:
I add a bigger one, and a slider to modify the x, height, width...